GetandTake/Configuration/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace GetandTake.Configuration;
+
+public static class CustomExceptionHandler
+{
+    public static void ExceptionHandler(this WebApplication app)
+    {
+        app.UseExceptionHandler(exceptionHandlerApp =>
+        {
+            exceptionHandlerApp.Run(async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status500InternalServerError;
+                context.Response.ContentType = Text.Plain;
+
+                await context.Response.WriteAsync("An exception was thrown.");
+
+                var exceptionHandlerPathFeature =
+                    context.Features.Get<IExceptionHandlerPathFeature>();
+                if (exceptionHandlerPathFeature != null)
+                {
+                    if (exceptionHandlerPathFeature?.Error is FileNotFoundException)
```

Comment:
Since you have added null check for `exceptionHandlerPathFeature` no need for `?` operator :)